src/Client/Search.php

```diff
@@ -17,7 +17,7 @@
 
 final class Search implements Iterator, Sequence
 {
-    const VERSION_SEARCH = 1;
+    const VERSION_SEARCH = 2;
```

Comment:
We cannot progress the version (although available in api-raml and api-dummy) until we progress in search application. We should prioritise that. Search should accept requests to version 1 and 2.